examples/ciao/ciao_cleanup.yml

```diff
@@ -0,0 +1,43 @@
+---
+- hosts: ciao_controller:ciao_compute:ciao_network
+  tasks:
+
+    # In ansible 2.2.0 we can drop ignore errors since it will not fail
+    # when the services does not exist.
+    - name: Stop services
+      service: name={{ item }} enabled=no state=stopped
+      with_items:
+        - ciao-network.service
+        - ciao-compute.service
+        - ciao-controller.service
+        - ciao-scheduler.service
+      ignore_errors: yes
+
+    - name: Remove CIAO Service files
+      file: name={{ item }} state=absent
+      with_items:
+        - /etc/systemd/system/ciao-network.service
+        - /etc/systemd/system/ciao-compute.service
+        - /etc/systemd/system/ciao-scheduler.service
+        - /etc/systemd/system/ciao-controller.service
+
+    - name: Remove CIAO directories
+      file: path={{ item }} state=absent
+      with_items:
+        - /etc/ca-certs
+        - /etc/ciao
+        - /etc/pki/ciao
+        - /etc/ssl/certs
```

Comment:
/etc/ssl/certs is a symlink to /etc/ca-certs

I dont want to delete /etc/ssl directory since the user might have a custom /etc/ssl/openssl.cnf